python/ray/scripts/scripts.py

```diff
@@ -1487,6 +1488,57 @@ def globalgc(address):
     print("Triggered gc.collect() on all workers.")
 
 
+@cli.command()
+@click.option("-v", "--verbose", is_flag=True)
+@click.option(
+    "--dryrun",
+    is_flag=True,
+    help="Identifies the wheel but does not execute the installation.")
+def install_nightly(verbose, dryrun):
+    """Install the latest wheels for Ray.
+
+    This uses the same python environment as the one that Ray is currently
+    installed in. Make sure that there is no Ray processes on this
+    machine (ray stop) when running this command.
+    """
+    raydir = os.path.abspath(os.path.dirname(ray.__file__))
+    all_wheels_path = os.path.join(raydir, "nightly-wheels.yaml")
+
+    wheels = None
+    if os.path.exists(all_wheels_path):
+        with open(all_wheels_path) as f:
+            wheels = yaml.safe_load(f)
+
+    if not wheels:
+        raise click.ClickException(f"Wheels not found in '{all_wheels_path}'!")
```

Comment:
Can we provide the same URL as below here too?

Reply:
This isn't a URL; it's a filepath.

Reply:
It's supposed to be on your local machine. The below path is the *specific S3 wheel path*.

Reply:
Oh what I meant was can we give them the same "Please visit https://docs.ray.io/en/master/installation.html to obtain the latest wheels."

Reply:
ah nice, done